python-stdlib/unittest-discover/unittest/__main__.py

```diff
@@ -121,22 +121,27 @@ def discover_main():
     ):
         # No args, or `python -m unittest discover ...`.
         result = _discover(runner)
-        failures += result.failuresNum or result.errorsNum
```

Comment:
Line 114 above with `failures = 0` can go.

Reply:
whoops, I thought I cleaned up the failures variable, good catch.